classes/PodsAdmin.php

```diff
@@ -2194,13 +2312,13 @@ public function admin_capabilities ( $capabilities ) {
                 }
             }
             elseif ( 'taxonomy' == $pod[ 'type' ] ) {
-                if ( 1 == pods_var( 'capabilities', $pod[ 'options' ], 0 ) ) {
+                if ( 'custom' == pods_var( 'capability_type', $pod[ 'options' ], 'terms' ) ) {
                     $capability_type = pods_var( 'capability_type_custom', $pod[ 'options' ], pods_var_raw( 'name', $pod ) . 's' );
 
-                    $capabilities[] = 'manage_' . $capability_type;
-                    $capabilities[] = 'edit_' . $capability_type;
-                    $capabilities[] = 'delete_' . $capability_type;
-                    $capabilities[] = 'assign_' . $capability_type;
+                    $capabilities[] = 'manage_' . $capability_type . '_terms';
```

Comment:
We add it to each cap here on each line, but in PodsInit.php we append $capability_type once